assets/src/edit-story/components/library/panes/pageLayouts/pageLayoutsIcon.js

```diff
@@ -23,10 +23,10 @@ import { __ } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
-import { Layout } from '../../../../../design-system/icons/editor';
+import { Icons } from '../../../../../design-system';
 
-const StyledLayoutIcon = styled(Layout)({
-  transform: 'scale(0.71)',
+const StyledLayoutIcon = styled(Icons.Box4Alternate)({
+  transform: 'scale(1.41)',
```

Comment:
This is a purely temporary value until #6207 is implemented. This will go out for 1.4, but should be fine.